example/integration_test/app_test.dart

```diff
@@ -0,0 +1,23 @@
+// Imports the Flutter Driver API.
+import 'package:purchases_flutter/purchases_flutter.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+    setUpAll(() async {
+      print("setupAll");
+      const apiKey = String.fromEnvironment("API_KEY", defaultValue: null);
+      await Purchases.setup(apiKey, appUserId: "test-user");
+    });
+
+  test("Configures successfully", () async {
+    expect(Purchases.appUserID != null, true);
```

Comment:
other than checking that it won't crash, does this actually confirm that it's a successful config? i.e.: if the communication with the backend fails, wouldn't this still work?

Reply:
the purchaserInfo test should catch most issues correctly, though. 
so maybe we just need to rename this to `Configures without crashing` or something to that effect